### PR TITLE
Bug/ts errors

### PR DESCRIPTION
**Pull request by gesseekur**
Fix TS errors

### CODE REVIEW

File: client/src/modules/uncles/components/TableUncles.vue [hunk 1]
```diff
@@ -81,7 +81,7 @@ import AppTableTitle from '@app/core/components/ui/AppTableTitle.vue'
 import TableUnclesRow from '@app/modules/uncles/components/TableUnclesRow.vue'
 import TableUnclesRowLoading from '@app/modules/uncles/components/TableUnclesRowLoading.vue'
 import { Vue, Component, Prop } from 'vue-property-decorator'
-import { UncleSummaryPageExt } from '@app/core/api/apollo/extensions/uncle-summary-page.ext'
+// import { UncleSummaryPageExt } from '@app/core/api/apollo/extensions/uncle-summary-page.ext'
```
Comment (olgakup) [inline client/src/modules/uncles/components/TableUncles.vue:84]: you can delete this

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue [hunk 2]
```diff
@@ -260,14 +260,14 @@ export default class TxDetails extends Mixins(NumberFormatMixin) {
      *
      * @return {String}
      */
-    get txFee(): string | null {
+    get txFee(): any {
```
Comment (olgakup) [inline newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue:263]: can you change this to:  'FormattedNumber' ?

File: client/src/modules/uncles/components/TableUncles.vue (hunk 1 above)
Comment (gesseekur) [reply client/src/modules/uncles/components/TableUncles.vue]: oops! I actually deleted this in the new client folder, and actually pushed this from client, I will remove this

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue (hunk 2 above)
Comment (gesseekur) [reply newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue]: changed!

**Pull request merged**